Binary files murrine-0.98.2.orig/src/.support.h.swp and murrine-0.98.2.problem/src/.support.h.swp differ
diff --exclude='*.*o' --exclude=.libs --exclude='*la' -ruN murrine-0.98.2.orig/src/murrine_rc_style.h murrine-0.98.2.problem/src/murrine_rc_style.h
--- murrine-0.98.2.orig/src/murrine_rc_style.h	2012-01-17 12:46:01.000000000 -0500
+++ murrine-0.98.2.problem/src/murrine_rc_style.h	2020-08-06 11:54:33.000000000 -0400
@@ -154,5 +154,7 @@
 };
 
 GType murrine_rc_style_get_type	(void);
+void
+murrine_rc_style_register_types (GTypeModule *module);
 
 #endif /* MURRINE_RC_STYLE_H */
diff --exclude='*.*o' --exclude=.libs --exclude='*la' -ruN murrine-0.98.2.orig/src/murrine_style.h murrine-0.98.2.problem/src/murrine_style.h
--- murrine-0.98.2.orig/src/murrine_style.h	2012-01-17 12:46:01.000000000 -0500
+++ murrine-0.98.2.problem/src/murrine_style.h	2020-08-06 11:53:52.000000000 -0400
@@ -102,5 +102,7 @@
 };
 
 GType murrine_style_get_type (void);
+void
+murrine_style_register_types (GTypeModule *module);
 
 #endif /* MURRINE_STYLE_H */
diff --exclude='*.*o' --exclude=.libs --exclude='*la' -ruN murrine-0.98.2.orig/src/support.h murrine-0.98.2.problem/src/support.h
--- murrine-0.98.2.orig/src/support.h	2012-01-17 12:46:01.000000000 -0500
+++ murrine-0.98.2.problem/src/support.h	2020-08-06 11:53:17.000000000 -0400
@@ -44,6 +44,11 @@
 #define MENU_OPACITY 0.90
 #define TOOLTIP_OPACITY 0.90
 
+gboolean murrine_widget_is_ltr (GtkWidget *widget);
+gboolean
+murrine_object_is_a (const GObject * object, const gchar * type_name);
+
+
 /* From gtk-engines 20071109 */
 #define MRN_IS_WIDGET(object) ((object)  && murrine_object_is_a ((GObject*)(object), "GtkWidget"))
 #define MRN_IS_CONTAINER(object) ((object)  && murrine_object_is_a ((GObject*)(object), "GtkContainer"))
